src/components/ThemeToggler.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div>
+    <input
+      @change="toggleTheme"
+      id="checkbox"
+      type="checkbox"
+      class="switch-checkbox"
+    />
+    <label for="checkbox" class="switch-label">
+      <span>🌙</span>
+      <span>☀️</span>
+      <div
+        class="switch-toggle"
+        :class="{ 'switch-toggle-checked': userTheme === 'dark-theme' }"
+      ></div>
+    </label>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+
+export default defineComponent({
+  name: "ThemeToggler",
```

Comment:
Can you rename this throughout as just `ThemeToggle` ?

Reply:
do you think I need to rename the file as ThemeToggle.vue as well?

Reply:
yes - do a find and replace in the code and rename the file

```sh
$ git mv ThemeToggler.vue ThemeToggle.vue
$ git commit
```